src/main/resources/application-test.properties

```diff
@@ -11,4 +11,16 @@ spring.datasource.continue-on-error=true
 insights.properties=dummy
 
 spring.datasource.url = jdbc:h2:mem:test
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.H2Dialect
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.H2Dialect
```

Comment:
Having the power of testcontainer available, i think it would be better to replace H2 database with PostgreSQL 9.6 so that the test will be using the same database used in the running environment.

Reply:
Especially given that it is as easy as:
```properties
spring.datasource.url=jdbc:tc:postgresql:9.6.8:///databasename
spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
```

:)

Reply:
thanks for the support @bsideup, really appreciated :+1:

Reply:
For the unit tests we will still use H2 and for the End2End we will use PostgreSQL